graph.rb

```diff
@@ -19,6 +22,15 @@ def topological_sort
     end
   end
 
+  def acyclic?
+    begin
```

Comment:
Methods implicitly act like `begin` blocks, so adding a new one here is unnecessary. We could write this method as:
```ruby
def cyclic?
  topological_sort
  false
rescue UnsortableCyclicGraphError
  true
end
```